site/content/docs/5.2/components/list-group.md

```diff
@@ -296,6 +321,31 @@ And if you want `<label>`s as the `.list-group-item` for large hit areas, you ca
 </div>
 {{< /example >}}
 
+{{< example >}}
```

Comment:
IMO in order to lighten the documentation, this example is not necessary. I'd say that the previous one with checkboxes already shows how it works.